src/app/(Authentication)/Login/Password/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import { passwordExists, signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const email = useLocalSearchParams() as unknown as string;
+  const [displayPassword, setDisplayPassword] = useState(false);
+  // const [isPasswordVisible, setPasswordVisible] = useState(false);
+
+  function inputPassword(password: string) {
```

Comment:
Same little bug/edge case as above but if we type then clear all text, the display is both above and on the line.